groups/bsl/bslstl/bslstl_inplace.h

```diff
@@ -9,26 +9,33 @@ BSLS_IDENT("$Id: $")
 //
 //@CLASSES:
 //  bsl::in_place_t: tag type for in-place construction
+//  bsl::in_place_type_t: tag type for in-place construction
+//  bsl::in_place_index_t: tag type for in-place construction
 //
 //@DESCRIPTION: This component provides an implementation of a standard
-// compliant tag type for in-place construction, 'bsl::in_place_t'.  This tag
-// type is used in constructors of 'bsl::optional' to indicate that the
-// contained object should be constructed in-place.  Note that the standard
-// currently has two other in-place construction tag types,
-// 'std::in_place_type_t' and 'std::in_place_index_t'. There is currently no
-// need to provide the equivalent tag types in bsl, but if such need arises,
-// the name of this header has been chosen for the purpose of being the
-// designated place for all in-place construction tags.
+// compliant tag types for in-place construction: 'bsl::in_place_t',
+// 'bsl::in_place_type_t', and 'bsl::in_place_index_t'. Tag type
+// 'bsl::in_place_t' is used in constructors of 'bsl::optional' to indicate
+// that the contained object should be constructed in-place.  Tag type
+// 'bsl::in_place_type_t<TYPE>' is used in constructors of 'bsl::variant' to
+// indicate that the object of type 'TYPE' should be constructed in-place.  Tag
+// type 'bsl::in_place_index_t<INDEX>' is used in constructors of
+// 'bsl::variant' to indicate that the alternative with index 'INDEX' should
+//  be constructed in-place.
+//
+//@SEE_ALSO: bslstl_optional, bslstl_variant
 
 #include <bslscm_version.h>
 
 #include <bsls_compilerfeatures.h>
 #include <bsls_keyword.h>
 #include <bsls_libraryfeatures.h>
 
+#include <stddef.h>
+
 #ifdef BSLS_LIBRARYFEATURES_HAS_CPP17_BASELINE_LIBRARY
 #include <utility>  // for std::in_place_t
-#endif  // BSLS_LIBRARYFEATURES_HAS_CPP17_BASELINE_LIBRARY
+#endif              // BSLS_LIBRARYFEATURES_HAS_CPP17_BASELINE_LIBRARY
```

Comment:
Please revert:
```suggestion
#endif  // BSLS_LIBRARYFEATURES_HAS_CPP17_BASELINE_LIBRARY
```
If bde-format made you format it like this, it's because of a known bug.

Reply:
i'll keep this conversation open in hope of catching it next time i run bde-format